script/hassfest/translations.py

```diff
@@ -227,6 +227,25 @@ def gen_strings_schema(config: Config, integration: Integration):
             vol.Optional("application_credentials"): {
                 vol.Optional("description"): cv.string_with_no_html,
             },
+            vol.Optional("issues"): {
+                str: vol.All(
+                    cv.has_at_least_one_key("description", "fix_flow"),
+                    vol.Schema(
+                        {
+                            vol.Required("title"): cv.string_with_no_html,
+                            vol.Exclusive(
+                                "description", "fixable"
```

Comment:
Should we not allow a description if the issue is fixable? Is the idea that the fix flow will describe the issue further?

Reply:
If there is a flow, you'll immediately drop into the flow (no description is shown)